automobile/sumo-interface.md

```diff
@@ -4,79 +4,131 @@ An interface with a microscopic traffic simulator called `SUMO` (Simulation of
-| --maximum-angular-speed  | Specifies the maximal angular speed of any vehicle in radian per second                                                              | By default, the maximum angular speed is 3 rad/s                                                 |
-| --lane-change-delay      | Specifies the time spent to change of lane (during this period, the position of the car in Webots and SUMO may not be synchronized anymore)          | By default, lane change takes 3 seconds                                                |
-| --sumo-arguments         | Specifies additional SUMO arguments                                                                                                  | By default, SUMO is started without any additional argument                                      |
-
-%end
 
+### The SumoInterface PROTO
 
-> **Note** [macOS]:
-On macOS, SUMO relies on X11. You need therefore to install [XQuartz](https://www.xquartz.org) (version 2.7.8 or later) for the interface to work.
+Here is the parameters of the `SumoInterface` PROTO (which inherits from the `Supervisor` node):
+
+```
+PROTO SumoInterface [
+  field SFBool    gui                   TRUE
+  field SFBool    useNetconvert         TRUE
+  field SFBool    enableTrafficLights   TRUE
+  field SFBool    enableWheelsRotation  FALSE
+  field SFInt32   maxVehicules          100
+  field SFInt32   radius               -1
+  field SFFloat   maximumLateralSpeed   2.5
+  field SFFloat   maximumAngularSpeed   3
+  field SFFloat   laneChangeDelay       3
+  field SFBool    enableHeight          FALSE
+  field SFInt32   step                  200
+  field SFInt32   port                  8873
+  field SFInt32   seed                  1
+  field SFString  networkFiles          ""
+  field SFString  sumoArguments         ""
+  field SFNode    display               NULL
+  field SFBool    verbose               FALSE
+  field SFBool    synchronization       TRUE
+  field MFNode    children              []
+]
+```
 
+#### SumoInterface fields summary
+
+- `gui`: Defines if the command-line or GUI version of SUMO should be used.
+- `useNetconvert`: Defines if NETCONVERT should be called before launching SUMO, this should be disabled if a `*.net` is already provided.
+- `enableTrafficLights`: Defines if the traffic lights synchronization should be used or not.
+- `enableWheelsRotation`: Defines if the wheels of the vehicles should rotate or not (wheels rotation has an impact on the simulation speed but make the simulation looks more realistic).
+- `maxVehicules`: Defines the maximum number of vehicles added in Webots from SUMO.
+- `radius`: Defines the visibility radius of the vehicles in meters (only vehicles closer to the viewpoint than this radius will be added in Webots). A negative value means that all the vehicles are added.
```

Comment:
The sentence in parenthesis sounds odd. I'd say "(if the distance between the viewpoint and a vehicle is greater than this radius, the vehicle will not added in Webots)".